R/dsl.R

```diff
@@ -430,10 +425,10 @@ menu_item <- function(..., item_feature = "", style = NULL, href = NULL) {
 #'
 #' @examples
 #'
-#' uidropdown(
+#' dropdown_menu(
```

Comment:
Please fix examples

Reply:
here again not sure what you're referring too, this example worked for me